haret/tests/utils/arbitrary.rs

```diff
@@ -42,10 +42,10 @@ impl Arbitrary for Op {
         let range = Range::new(0, 100);
         match range.ind_sample(&mut thread_rng()) {
             0...80 => Op::ClientRequest(ClientRequest::arbitrary(g)),
-            80...85 => Op::Commit,
```

Comment:
Yeah, it's a bit unfortunate. Here's a rather old [thread](https://internals.rust-lang.org/t/vs-for-inclusive-ranges/1539) talking about this. Unfortunately it's a bit too late to really do anything about it.